blueprints/ember-electron/index.js

```diff
@@ -24,96 +21,79 @@ class EmberElectronBlueprint extends Blueprint {
   afterInstall(/* options */) {
     let logger = new Logger(this);
 
-    return this._installDependencies(logger)
-      .then(() => this._createResourcesDirectories())
-      .then(() => this._ensurePackageJsonConfiguration())
-      .then(() => this._updateGitignore())
```

Comment:
To be fair, I don't think this should be considered blocking, as we're tracking in #189 & a lot of subsequent work will likely be contingent on this.